traffic_portal/package.json

```diff
@@ -50,12 +50,12 @@
     "angular-route": "1.8.0",
     "angular-sanitize": "1.8.0",
     "angular-ui-bootstrap": "0.14.3",
-    "bootstrap-sass": "^3.3.6",
-    "es5-shim": "^4.5.6",
-    "flot": "^2.1.6",
-    "font-awesome": "^4.5.0",
-    "jquery": "^2.1.4",
-    "jquery.flot.tooltip": "^0.9.0",
-    "json3": "^3.3.2"
+    "bootstrap-sass": "3.3.6",
+    "es5-shim": "4.5.6",
+    "flot": "2.1.6",
```

Comment:
This downgrades `flot` from `2.3.2`, since `^` can refer to future minor or patch versions. The exact version should refer to what was previously the calculated `^` version to keep it from downgrading. So, `flot` should be `2.3.2`.

Reply:
Good catch, however when I added flot to `package.json` I should've used exact versioning to follow what we do everywhere else (or at least only allow patch release updates). I'll upgrade it to `2.3.2` since that's what `package-lock` was installing anyways.